shell/key-bindings.fish

```diff
@@ -1,7 +1,9 @@
 # Key bindings
 # ------------
 function fzf_key_bindings
-
+  set -g __FZF_FISH_MAJOR (echo $FISH_VERSION | cut -f1 -d.)
```

Comment:
Ok. I had expected people would prefer a global variable to unnecessary computation, but I'm happen with either.